recipe/meta.yaml

```diff
@@ -3,7 +3,7 @@
 {% set build_number = 0 %}
 
 package:
-  name: '{{ name|lower }}'
+  name: pymedphys-split
```

Comment:
Is this how other people name these kinds of packages? My preference is to just call it "pymedphys" if possible. 99% of users will always install just the top level package. The subpackages are to support environments where certain dependencies are not available.

Reply:
This name is a dummy name. There'll be no package named `pymedphys-split`. Since you have an output named `pymedphys`, there'll be a package named `pymedphys`

Reply:
Oh? is this fixing my issue? Re-running now.

Reply:
> This name is a dummy name. There'll be no package named `pymedphys-split`. Since you have an output named `pymedphys`, there'll be a package named `pymedphys`

Cheers :)

Reply:
That has certainly fixed the issue locally. ... now for the moment of truth... does it pass on Azure ... suspense ...